specs/core/0_beacon-chain.md

```diff
@@ -226,54 +232,58 @@ Unless otherwise indicated, code appearing in `this style` is to be interpreted
 
 ## Data structures
 
-### Deposits
+### Beacon chain operations
+
+#### Proposer slashings
 
-#### `DepositParametersRecord`
+##### `ProposerSlashing`
 
 ```python
 {
-    # BLS pubkey
-    'pubkey': 'uint384',
-    # BLS proof of possession (a BLS signature)
-    'proof_of_possession': ['uint384'],
-    # Withdrawal credentials
-    'withdrawal_credentials': 'hash32',
-    # Initial RANDAO commitment
-    'randao_commitment': 'hash32',
+    # Proposer index
+    'proposer_index': 'uint24',
+    # First proposal data
+    'proposal_data_1': ProposalSignedData,
+    # First proposal signature
+    'proposal_signature_1': '[uint384]',
+    # Second proposal data
+    'proposal_data_2': ProposalSignedData,
+    # Second proposal signature
+    'proposal_signature_2': '[uint384]',
 }
 ```
 
-### Beacon chain blocks
+#### Casper slashings
 
-#### `BeaconBlock`
+##### `CasperSlashing`
 
 ```python
 {
-    ## Header ##
-    'slot': 'uint64',
-    # Skip list of ancestor beacon block hashes
-    # i'th item is the most recent ancestor whose slot is a multiple of 2**i for i = 0, ..., 31
-    'ancestor_hashes': ['hash32'],
-    'state_root': 'hash32',
-    'randao_reveal': 'hash32',
-    'candidate_pow_receipt_root': 'hash32',
-    'signature': ['uint384'],
-
-    ## Body ##
-    'body': BeaconBlockBody,
+    # First batch of votes
+    'votes_1': CasperVotes,
+    # Second batch of votes
+    'votes_2': CasperVotes,
 }
 ```
 
-#### `BeaconBlockBody`
+##### `CasperVotes`
```

Comment:
IMO plural `Votes` is confusing sometimes. Per @djrtwo: `SlashableAttestationData`? Or something like `VoteData`?